examples/Boost-thread/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2013, Ruslan Baratov
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-boost)
+
+# download boost
+hunter_add_package(Boost COMPONENTS system thread)
+
+find_package( Threads )
```

Comment:
There is no explicit use of this package. If it's needed for Boost then it should be done in `BoostConfig.cmake`.